influxql/v1tests/server_test.go

```diff
@@ -23,7 +23,6 @@ const (
 
 // Ensure the server can query with default databases (via param) and default retention policy
 func TestServer_Query_DefaultDBAndRP(t *testing.T) {
-	t.Parallel()
```

Comment:
For some reason, the `t.Parallel`s here were nuking Circle. Removing them didn't significantly impact the total runtime of the `test` job: the latest merge to `master` took `8m28s`, and the run from the commit that removed all these `Parallel` calls took `7m11s`